packages/pglite/src/live/index.ts

```diff
@@ -33,9 +33,9 @@ const setup = async (pg: PGliteInterface, emscriptenOpts: any) => {
       const init = async () => {
         await pg.transaction(async (tx) => {
           // Create a temporary view with the query
+          const formattedQuery = await formatQuery(tx, query, params || []);
```

Comment:
```suggestion
          const formattedQuery = await formatQuery(tx, query, params);
```

Should work even if it is null or undefined based on the utility